File: xiajiwangluo_store/src/main/webapp/js/service/inoutService.js
//服务层
app.service('inoutService',function($http){

    //查询入库明细
    this.findAll=function(cls){
        return $http.get('../inout/findAll.do?cls='+cls);
    };

    //库存查询
    this.findPage=function (pageNum,pageSize,da_list){
        return $http.post('../inout/findPage.do?pageNum='+pageNum+'&pageSize='+pageSize, da_list);
    }
});

File: xiajiwangluo_store/src/main/java/com/xiajiwangluo/service/InoutService.java
package com.xiajiwangluo.service;

import com.xiajiwangluo.bean.Inout;
import com.xiajiwangluo.bean.beangroup.DA_list;
import com.xiajiwangluo.bean.entity.PageResult;

import java.util.List;

public interface InoutService {

    List<Inout> findAll(String sid,Integer cls);

    /***
     * 分页
     * @param pageNum 当前页面
     * @param pageSize 记录数
     * @return
     */
    public PageResult findPage(int pageNum, int pageSize , DA_list da_list);
}
